Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -395,7 +395,7 @@ public struct ConvertAction: Action, RecreatingContext {
         }
 
         // Log the output size.
-        benchmark(add: Benchmark.OutputSize(dataURL: targetDirectory.appendingPathComponent(NodeURLGenerator.Path.dataFolderName)))
+        benchmark(add: Benchmark.OutputSize(dataURL: targetDirectory))
```

Comment:
Is this change so that we track the size of other files such as the index as part of these metrics?

Reply:
Ah actually I see that this is a cherry-pick of https://github.com/apple/swift-docc/pull/122